pybb/defaults.py

```diff
@@ -113,6 +113,7 @@ def getsetting_with_deprecation_check(all_settings, setting_name):
 
 PYBB_DISABLE_SUBSCRIPTIONS = getattr(settings, 'PYBB_DISABLE_SUBSCRIPTIONS', False)
 PYBB_DISABLE_NOTIFICATIONS = getattr(settings, 'PYBB_DISABLE_NOTIFICATIONS', False)
+PYBB_NOTIFY_ON_EDIT = getattr(settings, 'PYBB_NOTIFY_ON_EDIT', False)
```

Comment:
You should have `True` as default value to stay fully backward compatible as in current version we are notified each time a post is updated.